web/test/pages/profile/profile-owning.test.tsx

```diff
@@ -496,6 +617,141 @@ describe("profile - owning existing business", () => {
         within(profileAlert).getByText(Config.profileDefaults.fields.sectorId.default.header)
       ).toBeInTheDocument();
     });
+
+    it("displays alert with city and zip code when blur on completed address line 1", async () => {
+      const business = generateBusinessForProfile({
+        profileData: generateProfileData({
+          industryId: "generic",
+          businessPersona: "OWNING",
+          operatingPhase: "UP_AND_RUNNING_OWNING",
+        }),
+        formationData: generateFormationData({
+          formationFormData: generateFormationFormData({
+            ...emptyAddressData,
+          }),
+        }),
+      });
```

Comment:
[dust] Given how often we're repeating the same logic in the business setup, it might be nice to see this pulled out to be included in a `beforeEach` or setup a generator within the scope of this describe block that could be used to just override the portions we need.